### PR TITLE
Introduce marker size component

### PR DESCRIPTION
**Pull request by Wumpf**
### What

* Needed for #5042

Allows to configure size again for points wiht the new PointSeries archetype.

Makes size of plot line and marker independent which is kinda neat :)
image
(Color is *not* independent, we'll want to solve this differently in the future in accordance to the model we're going with, but up for discussion obviously)

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5057/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5057/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5057/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/5057)
- [Docs preview](https://rerun.io/preview/6c716760a61522d8f5638b0aad12a2dd84064e01/docs) 
- [Examples preview](https://rerun.io/preview/6c716760a61522d8f5638b0aad12a2dd84064e01/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: crates/re_space_view_time_series/src/lib.rs [hunk 1]
```diff
@@ -47,12 +47,12 @@ impl PartialEq for PlotPointAttrs {
         let Self {
             label,
             color,
-            stroke_width: radius,
+            marker_size: radius,
```
Comment (teh-cmc) [inline crates/re_space_view_time_series/src/lib.rs:50]:
```suggestion
            marker_size,

File: crates/re_space_view_time_series/src/line_visualizer_system.rs [hunk 2]
```diff
@@ -141,16 +140,19 @@ impl SeriesLineSystem {
                 let query = re_data_store::RangeQuery::new(query.timeline, time_range);
 
                 // TODO(jleibs): need to do a "joined" archetype query
+                // The `Scalar` archetype queries for `MarkerSize` in the point visualizer,
+                // and so it must do so here also.
+                // See https://github.com/rerun-io/rerun/pull/5029
```
Comment (teh-cmc) [inline crates/re_space_view_time_series/src/line_visualizer_system.rs:145]: merge this note with the already existing note below

File: crates/re_types/definitions/rerun/archetypes/series_point.fbs [hunk 3]
```diff
@@ -26,4 +26,9 @@ table SeriesPoint (
     ///
     /// Used in the legend.
     name: rerun.components.Name ("attr.rerun.component_optional", nullable, order: 3000);
+
+    /// Size of the markers.
+    ///
+    /// Can set a single one for all markers or a batch of sizes.
```
Comment (teh-cmc) [inline crates/re_types/definitions/rerun/archetypes/series_point.fbs:32]: Isn't it declared as a mono right below? How could it be a batch?

**Review by teh-cmc — APPROVED** [review]

**Comment by teh-cmc** [comment]
- Checked on web :white_check_mark:

File: crates/re_types/definitions/rerun/archetypes/series_point.fbs (hunk 3 above)
Comment (Wumpf) [reply crates/re_types/definitions/rerun/archetypes/series_point.fbs:32]: urgh, thanks. Was too hasty!

**Pull request merged**